dev-docs/modules/audigentRtdProvider.md

```diff
@@ -0,0 +1,91 @@
+---
+layout: page_v2
+title: Audigent Segmentation RTD Module
+description: Audigent Segmentation RTD Module
+module_code : audigent
```

Comment:
Please add the following header metadata so this module will appear in the modules list and download page:

- display_name: Audigent Segmentation
- module_type : rtd